lib/LaTeXML/Package/pgfmath.code.tex.ltxml

```diff
@@ -468,23 +470,23 @@ BEGIN {
     or         => sub { $_[0] || $_[1]; },
     '+'        => sub { (defined $_[1] ? $_[0] + $_[1] : $_[0]); },
     'add'      => sub { (defined $_[1] ? $_[0] + $_[1] : $_[0]); },
-    '-'        => sub { (defined $_[1] ? $_[0] - $_[1] : -$_[0]); },    # prefix or infix
+    '-'        => sub { (defined $_[1] ? $_[0] - $_[1] : -$_[0]); },                   # prefix or infix
     neg        => sub { -$_[0]; },
     '*'        => sub { $_[0] * $_[1]; },
     multiply   => sub { $_[0] * $_[1]; },
-    '/'        => sub { $_[0] / $_[1]; },
-    div        => sub { int($_[0] / $_[1]); },
-    divide     => sub { $_[0] / $_[1]; },
-    '!'        => sub { pgfmathfactorial($_[0]); },
-    'r'        => sub { rad2deg($_[0]); },
-    e          => sub { $E; },
-    pi         => sub { $PI; },
-    abs        => sub { abs($_[0]); },
-    acos       => sub { acos($_[0]); },
-    array      => sub { },
-    asin       => sub { rad2deg(asin($_[0])); },
-    atan       => sub { rad2deg(atan($_[0])); },
-    atan2      => sub { rad2deg(atan2($_[0], $_[1])); },
+    '/'        => sub { my $divisor = (0.0 + $_[1]) || $epsilon; $_[0] / $divisor; },
+    divide     => sub { my $divisor = (0.0 + $_[1]) || $epsilon; $_[0] / $divisor; },
+    div   => sub { my $divisor = (0.0 + $_[1]) || $epsilon; int($_[0] / $divisor); },
```

Comment:
I'm all for avoiding fatals, but, again, skipping over it silently seems dangerous for non-arXiv usecases.

Reply:
Well, it won't be silent - the cases where a zero sneaks in almost invariably have a register warning of `expected:<number>` with `Missing number, treated as zero` details string.

Reply:
In arXiv, where you only find valid TeX (cough), sure; divide by zero is probably due to some LaTeXML register mishandling that "assumed 0". But in general usage, it might show up while somebody's tweaking code for a graphic.

Reply:
Here's a baseline TeX mistake where division by 0 happens:
```tex
\documentclass[tikz]{standalone}
\begin{document}
\begin{tikzpicture}
  \draw (0,1) node {\pgfmathparse{1/0}\pgfmathresult\ result prints ok} ;
\end{tikzpicture}
\end{document}
```

pdflatex will print:
```
! Package PGF Math Error: You'veaskedmetodivide`1'by`0',butIcannotdivideanynumb
erby`0' (in '1/0').
```

latexml master will Fatal. This PR will produce a broken rendering with "no problem" status. I suppose I can shoot off a warning akin to this PGF Math Error, so that a trace is preserved...